.github/workflows/build.yml

```diff
@@ -9,8 +9,10 @@ jobs:
       - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
         with:
           submodules: true
+      - name: install python3-requests
+        run: pip3 install requests
```

Comment:
I tought this was added to dockers itself, do we need it here?

Reply:
This was added to `rust_build_utils` docker. These are the github's VMs, which is provisioned separately.